jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/OdeSsmDataCreatorHelperTest.java

```diff
@@ -1,40 +1,45 @@
 package us.dot.its.jpo.ode.coder;
 
+import static net.javacrumbs.jsonunit.JsonMatchers.jsonEquals;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
-
-import org.junit.jupiter.api.Test;
 
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Test;
 import us.dot.its.jpo.ode.model.OdeSsmData;
-import us.dot.its.jpo.ode.util.XmlUtils;
-import us.dot.its.jpo.ode.util.XmlUtils.XmlUtilsException;
 
+/**
+ * Unit tests for the OdeSsmDataCreatorHelper class.
+ */
```

Comment:
Mainly checkstyle updates to this unit test. Moving sample data to dedicated files and javadocs.